File: tests/test3.py
from timeit import default_timer as timer
class test3:
    def main(self):
        self.testFunc()
    def testFunc(self):
        e = len('hello')
        for i in range(10000):
            print(i)
            print(e)
if __name__ == '__main__':
    start = timer()
    test3().main()
    end = timer()
    print(end-start)

File: tests/test3.java
public class test3
{
    public static void main(String[] args) 
    {
        testFunc();
    }
    public static void testFunc(){
        int e = ("hello").length();
        for (int i=0; i<10000; i = i+1){
            System.out.println(i);
            System.out.println(e);
        }
    }
} 